ProcessLib/SmallDeformation/SmallDeformationProcess-impl.h

```diff
@@ -69,6 +71,13 @@ void SmallDeformationProcess<DisplacementDim>::initializeConcreteProcess(
             NumLib::ComponentOrder::BY_LOCATION);
     _nodal_forces->resize(DisplacementDim * mesh.getNumberOfNodes());
 
+    _material_forces->resize(DisplacementDim * mesh.getNumberOfNodes());
+
+    Base::_secondary_variables.addSecondaryVariable(
+        "free_energy_density",
+        makeExtrapolator(1, getExtrapolator(), _local_assemblers,
+                         &LocalAssemblerInterface::getIntPtFreeEnergyDensity));
+
     Base::_secondary_variables.addSecondaryVariable(
         "sigma",
```

Comment:
if one name is "free_energy_density" then the other name should be "stress" instead of "sigma". But since sigma is a very widespread symbol for stress, I'm also ok with that.